edgelet/hsm-sys/azure-iot-hsm-c/src/hsm_log.c

```diff
@@ -30,5 +66,30 @@ void log_msg(int level, const char* file, const char* function, int line, const
         vsnprintf(buffer, MAX_LOG_SIZE, fmt_str, args);
         printf("<%d>%s [%s] (%s:%s:%d) %s\r\n", syslog_levels[level], time_buf, levels[level], file, function, line, buffer);
         va_end (args);
+
+#if defined __WINDOWS__ || defined _WIN32 || defined _WIN64 || defined _Windows
+        if (event_log_handle != NULL) {
+            size_t log_length = strlen(file) + strlen(function) + strlen(buffer) + 50;
```

Comment:
Can you please name the magic number with rationale as to how you arrived at it?

Reply:
I wish C had a better way of finding the expected length of a format string. that would be cool.

Reply:
Done.